### DISCUSSION TITLE
Upgrade to latest version of filebrowser

### DISCUSSION THREAD

**Discussion by valholm** [filebrowser/filebrowser#2686] [General]
Hello 

First of all - Love the this litte program :)

I have currently installed version 2.22.4 on a linux (Ubuntu) server. It's running fine also..
But I want to upgrade to latest version and I also don't want to brake the current version/settings/database in the process.

Currently i'm simply running the program as a command that i enter after server has been restarted.
_sudo filebrowser -r /home/ftp/ -d /home/ftp/xxx/filebrowser.db &_

Have looked into the documentation on filebrowser.org, but cannot clearly find a good explanation regarding how to do the upgrade other than the command for the quick install. But the quick install makes a new database as I understand it.

Hope to get some input on how to upgrade and still keep all my current settings/database..

Ed

**Comment by valholm** [comment]
Anyone able to assist?

Ed

**Reply by fnafox** [reply]
Replace the old filebrowser binary (you can use the command `which filebrowser` if you don't know where it is) with the new one you downloaded from the [releases page](https://github.com/filebrowser/filebrowser/releases). Once replaced, shut down the running filebrowser instance and start a new one.

**Comment by niubility000** [comment]
the quick install makes a new database as I understand it.

No， you still use the old database.

**Comment by c-michail** [comment]
This doesn't work with 2.31. I get bad gateway (i use nginx reverse proxy)

**Reply by c-michail** [reply]
needed to "chmod +x filebrowser" executable after replacing in order to work!
